client/src/App.js

```diff
@@ -450,16 +457,15 @@ class App extends Component {
       case ALUMNI:
         return (
           <>
+          <Navbar
+              userDetails={this.state.userDetails}
+              role={role}
+              timezoneActive={true}
+              navItems={alumniNavBarItems(this.state.approved, this.state.newRequestsCount, this.state.unseenMessagesCount)}
+          />
           <Route exact path={`/`} render={(props) => 
                   this.state.loggedIn ?
```

Comment:
nit for future improvement I don't think we should be checking logged in throughout like this. we could move it up and centralise it more

Reply:
Yeah I was thinking this as well, just didn't think it was worth fixing as part of this PR

Reply:
Yeah I'm pretty anti-Redux personally, but I think the Context API is definitely a good move in places for sure